sys-power/apcupsd/apcupsd-3.14.14-r4.ebuild

```diff
@@ -45,6 +45,11 @@ pkg_setup() {
 	fi
 }
 
+src_prepare() {
+	default
+	sed -i "/^SUBDIRS/ s/doc//g" Makefile || die
```

Comment:
Could do with a better comment about why it's safe to delete this.

Reply:
On  the bug is was commented that the doc step (which needs col) actually creates files that later are never installed. I can augment the patch to include this.